src/common/mixin.js

```diff
@@ -12,6 +17,7 @@ export default {
       default: true,
       type: Boolean,
     },
+    dateFormat: { String, default: 'YYYY-MM-DD' },
```

Comment:
Is this a new writing rule for the `props` options of vue? If not, please correct it

Reply:
it's corrected. thanks

Reply:
I think it is `{ type: String, default: 'YYYY-MM-DD' }`, isn't it?